File: 240421/두 개의 동일한 수열/two-equal-series.py
N = int(input())
A = list(map(int,input().split()))
B = list(map(int,input().split()))
A.sort()
B.sort()

is_same = True

for a,b in zip(A,B):
    if a != b:
        is_same = False
        break

if is_same:
    print("Yes")
else:
    print("No")

File: 240603/두 개의 동일한 수열/two-equal-series.java
import java.util.*;
import java.util.stream.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        List<Integer> listA = Arrays.stream(br.readLine().split(" "))
            .map(Integer::parseInt)
            .collect(Collectors.toList());
        List<Integer> listB = Arrays.stream(br.readLine().split(" "))
            .map(Integer::parseInt)
            .collect(Collectors.toList());
        
        listA.sort(Comparator.naturalOrder());
        listB.sort(Comparator.naturalOrder());

        for (int i=0; i<n; i++) {
            if (listA.get(i) != listB.get(i)) {
                System.out.println("No");
                return;
            }
        }
        System.out.println("Yes");
    }
}